runtime/compiler/env/J9CompilerEnv.cpp

```diff
@@ -46,8 +46,16 @@ J9::CompilerEnv::initializeTargetEnvironment()
    OMR::CompilerEnvConnector::initializeTargetEnvironment();
    }
 
+void
+J9::CompilerEnv::initializeRelocatableTargetEnvironment()
+   {
+   OMR::CompilerEnvConnector::initializeRelocatableTargetEnvironment();
+   if (J9_ARE_ANY_BITS_SET(javaVM->extendedRuntimeFlags2, J9_EXTENDED_RUNTIME2_ENABLE_PORTABLE_SHARED_CACHE))
```

Comment:
I verified that at the time this is called, the `J9_EXTENDED_RUNTIME2_ENABLE_PORTABLE_SHARED_CACHE` does get set.